crates/sui-core/src/authority_active.rs

```diff
@@ -361,7 +360,7 @@ where
         let node_sync_store = self.state.node_sync_store.clone();
 
         info!("spawning node sync task");
-        let join_handle = tokio::task::spawn(node_sync_process(
+        let join_handle = spawn_monitored_task!(node_sync_process(
```

Comment:
Does this introduce significant overhead?

Reply:
I don't think so. It adds a string format, and a gauge increment/decrement around the wrapped future.